client/state/form/actions.js

```diff
@@ -15,67 +16,47 @@ export const setFormProperty = ( field, value ) => {
 	};
 };
 
-export const goToStep = ( stepIndex ) => {
-	return {
-		type: GO_TO_STEP,
-		step: stepIndex,
-	};
-};
-
-export const nextStep = () => ( dispatch, getState, { callbackURL, nonce, submitMethod, formSchema, formLayout } ) => {
-	const submitForm = ( callback ) => {
-		const setIsSaving = ( value ) => {
-			dispatch( setFormProperty( 'isSaving', value ) );
-			if ( false === value ) {
-				callback();
-			}
-		};
-		const setSuccess = ( value ) => dispatch( setFormProperty( 'success', value ) );
-		const setError = ( value ) => dispatch( setFormProperty( 'errors', value ) );
-
-		dispatch( setFormProperty( 'pristine', true ) );
-		saveForm( setIsSaving, setSuccess, setError, callbackURL, nonce, submitMethod, getState().form.values );
-	};
-
-	const tryAutoAdvance = ( isManualAction ) => {
-		const currentStep = getState().form.currentStep;
-		const currentStepLayout = formLayout[ currentStep ] || {};
+export const backFromSuggestion = () => ( {
```

Comment:
What does `backFromSuggestion` mean? I think we need a more descriptive name here.